pkg/config/repoconfig/repoconfig.go

```diff
@@ -15,14 +15,43 @@ const (
 )
 
 type BotConfig struct {
-	LabelApproved *struct {
-		Approvals *int
-		Label     *string
-	} `yaml:"label-approved"`
+	LabelApproved *LabelApprovedConfig `yaml:"label-approved"`
 }
 
-func Read(gitRepo *git.Git, sha string) (*BotConfig, error) {
+type LabelApprovedConfig struct {
+	Approvals *int
+	Label     *string
+}
+
+func (c *BotConfig) HasLabelApproved() bool {
+	return c.LabelApproved != nil
+}
+
+func (c *BotConfig) GetLabelApproved() *LabelApprovedConfig {
+	if c.HasLabelApproved() {
+		return c.LabelApproved
+	}
+
+	return &LabelApprovedConfig{}
+}
+
+func (c *LabelApprovedConfig) GetApprovals() int {
+	if c.Approvals != nil {
+		return *c.Approvals
+	}
+
+	return defaultApprovals
```

Comment:
```suggestion
	if c.Approvals == nil {
		return defaultApprovals
	}

	return *c.Approvals
```

Main body should be the "expected path", exceptional paths branch out